SIL.Windows.Forms/ClearShare/MetaData.cs

```diff
@@ -361,7 +362,7 @@ public bool FileFormatSupportsMetadata(string path)
 			return file != null && !file.GetType().FullName.Contains("NoMetadata");
 		}
 
-		public void Write(string path)
+		public void Write(string path, string originalImagePath = null)
```

Comment:
I kind of agree it's awkward. But I think we need more control than you are suggesting, if we do this at all. Not every ClearShare.metadata write should copy all the TagLib.metadata from the image used to create it. For example, if we apply the same ClearShare.metadata to every file in a bloom book, doing that would make them all appear to have been taken in the same place at the same time, etc.
I suppose I can give Write a boolean which says whether to copy the original metadata and default it false...